### ISSUE TITLE
Reset Undownloadables

### ISSUE THREAD

**Issue by sirEgghead** [tubearchivist/tubearchivist#662]
I had an internet outage that caused my queue to get hung up.

> ERROR: [youtube] xuaF1Nsd7KA: Unable to download API page: <urllib3.connection.HTTPSConnection object at 0x7f99ce655d10>: Failed to establish a new connection: [Errno 113] No route to host (caused by TransportError('<urllib3.connection.HTTPSConnection object at 0x7f99ce655d10>: Failed to establish a new connection: [Errno 113] No route to host'))

I have over 300 videos in queue with that error in red.  I'm uncertain of how to force TA to try them all again.  I can successfully click "download now" on them individually, but that's a bit much.  I've tried ignoring or deleting and hoping to get them readded, but I was unsuccessful in those tests.

How can I force them all to try again?  Thanks!

**Comment by sirEgghead** [comment]
@lamusmaser thank you for the swift reply.  I have restarted TA (I did so again just now for continuity's sake).  The issue remains the same.  I can individually click "download now" and successfully download the single video, which confirms that my network and internet connection has been restored.  I haven't been able to force TA to recheck the ones that previously received an error during my internet outage.
